cvs -z9 diff listctrl.cpp (in directory E:\Dev\wxWindows\src\msw\)
Index: listctrl.cpp
===================================================================
RCS file: /home/wxcvs/wxWindows/src/msw/listctrl.cpp,v
retrieving revision 1.87
diff -r1.87 listctrl.cpp
452a453
> #if defined(_WIN32_IE) && (_WIN32_IE >= 0x0300)
456a458
> #endif


 	  	 
